### ISSUE TITLE
NPM Version Requirements Problem

### ISSUE THREAD

**Issue by darethehair** [erkstruwe/chromecast-cli#14]
Warnings installing on my current LMDE system:

```
darren@dell3010:~$ sudo npm install -g chromecast-cli
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'chromecast-cli@3.0.0',
npm WARN EBADENGINE   required: { node: '>=18', npm: '>=10' },
npm WARN EBADENGINE   current: { node: 'v18.19.0', npm: '9.2.0' }
npm WARN EBADENGINE }

changed 25 packages in 5s
```
I note that this does not (?) appear to prevent installation of 'chromecast-cli'.

**Comment by erkstruwe** [comment]
Hm, seems to me that some version of Node 18 are distributed with npm 9 and some with npm 10. Will change that. It is correct that these warnings don't prevent npm from installing the package.

**Comment by erkstruwe** [comment]
Fixed in ae71947d6c4736c0fefbd0d06db9235f6a20fe14 (released version 3.0.1).
